packages/web3-eth-contract/test/integration/contract_deploy.test.ts

```diff
@@ -148,7 +149,7 @@ describe('contract', () => {
 
 		it('should emit the "confirmation" event', async () => {
 			const confirmationHandler = jest.fn();
-
+			contract.setConfig({ transactionConfirmationBlocks: 2 });
```

Comment:
This is good, I think. And would not the value `1`, if set to `transactionConfirmationBlocks` makes things even faster? 

```suggestion
			contract.setConfig({ transactionConfirmationBlocks: 1 });
```

Reply:
done